packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -252,28 +252,28 @@ export class DebugConfigurationManager {
         return this.models.values().next().value;
     }
 
-    protected async doOpen(model: DebugConfigurationModel): Promise<EditorWidget> {
+    protected async doOpen(model: DebugConfigurationModel): Promise<EditorWidget | undefined> {
```

Comment:
Should we document this breakage to be safe?
A downstream extension may previously expect `EditorWidget` when calling `doOpen`.

Reply:
Yes, likely best to be safe. I'll push a changelog entry